cray_product_catalog/catalog_update.py

```diff
@@ -65,7 +65,8 @@
 PRODUCT_VERSION = os.environ.get("PRODUCT_VERSION").strip()  # required
 CONFIG_MAP = os.environ.get("CONFIG_MAP", "cray-product-catalog").strip()
 CONFIG_MAP_NAMESPACE = os.environ.get("CONFIG_MAP_NAMESPACE", "services").strip()
-YAML_CONTENT = os.environ.get("YAML_CONTENT").strip()  # required
+YAML_CONTENT = os.environ.get("YAML_CONTENT", "").strip()  # One of (YAML_CONTENT, RAW_YAML_CONTENT) required.
```

Comment:
I feel like YAML_CONTENT_FILE and YAML_CONTENT_STRING would be clearer environment variable names rather than YAML_CONTENT and RAW_YAML_CONTENT.

Reply:
I'd prefer to keep `YAML_CONTENT` for backwards compatibility but I could introduce `YAML_CONTENT_FILE` as an alternate name. Happy to rename `RAW_YAML_FILE` to `YAML_CONTENT_STRING`.

Reply:
I went ahead with this change. While renaming the variables I found some parts of `README.md` that needed updates.